README.md

```diff
@@ -148,6 +148,10 @@ for all files in `src` directory that match the glob pattern
 To package your app into an installer use command:
 
 ```sh
+npm install --production
```

Comment:
This isn't necessary: we're using `yarn` to manage `node_modules` instead of `npm`.